packages/react-ai/src/components/AIConversation/context/InputContext.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-interface Input {
+export interface Input {
```

Comment:
`Input` has a ton of usage as a UI component, can we rename please?

Reply:
Will rename